en/02_quicktour.md

```diff
@@ -0,0 +1,276 @@
+# 2. A Quick Tour
+
+## Sample for running SQL
+
+Since we have the sbt project set up, we are ready to run SQL using ScalikeJDBC. In this example I will use the [H2 Database](http://www.h2database.com/).
+
+Try running on the sbt console by copying the following code examples. Please note that the content of this book is available on GitHub, which you can also refer to;
+
+[Https://github.com/scalikejdbc/scalikejdbc-cookbook](https://github.com/scalikejdbc/scalikejdbc-cookbook)
+
+### Initializing a connection pool
+
+Firstly, here is how to load the JDBC driver and initialize a connection pool.
+
+    import scalikejdbc._
+    Class.forName("org.h2.Driver")
+    ConnectionPool.singleton("jdbc:h2:mem:scalikejdbc","user","pass")
+
+### Executing DDL
+
+We don't have a table yet, so create a `members` table by running the following. If no exception occurs, we have successfully created a table.
+
+    DB autoCommit { implicit session =>
+      SQL("""
+        create table members (
+          id bigint primary key auto_increment,
+          name varchar(30) not null,
+          description varchar(1000),
+          birthday date,
+          created_at timestamp not null
+        )
+      """).execute.apply()
+    }
+
+### Meaning of implicit session
+
+Here we saw something like `{implicit session => }`. This might confuse some of you who are not familiar with Scala because the variable `session` does not seem to be used anywhere else. I will briefly explain this.
+
+Firstly, `DB.autoCommit[A](...)` is a method that takes a function of type `(DBSession) => A` as its argument. It's normally called in this manner;
+
+    DB autoCommit { session =>
+    }
+
+Secondly, if we add `implicit` to the `session`, it becomes an implicit parameter just like a variable declared as `implicit val` in this scope. That is to say, this piece of code;
+
+    DB autoCommit { implicit session =>
+    }
+
+is equivalent to the following;
+
+    DB autoCommit { session =>
+     implicit val _session: DBSession = session
+    }
+
+So why did `session` have to be an implicit paramter? That's because the `apply` method as in `SQL("...").execute.apply()` we saw in the DB block implicitly expects a `DBSession` type parameter.
```

Comment:
s/paramter/parameter/

The reason the `session` should be an implicit parameter there is that the `apply` method of `SQL("...").execute.apply() ` in the DB block expects ...